steps/mobile-quality/1.0.0/step.yml

```diff
@@ -0,0 +1,181 @@
+title: steps-mobile-quality
+summary: |
+  Check and monitor mobile application's quality
+description: |
+  Check mobile application's quality, it can be an APK and/or IPA file. Among other things, you can:
+  - setup a config file, or directly on step's input, the value related to released app to compare the values on each new release candidate .eg, to be aware on each new permission and/or app size increased
+  - generate a report file where we will display the result of different check
+
+  ### Troubleshooting
+
+  - Be aware, you have to launch this step after the compilation step ("Android Build" for android, "Xcode Archive" for iOS)
+  - Launch "Deploy to Bitrise.io" step after this one to deploy report files to artifacts
+website: https://github.com/imranMnts/bitrise-step-mobile-quality
+source_code_url: https://github.com/imranMnts/bitrise-step-mobile-quality
+support_url: https://github.com/imranMnts/bitrise-step-mobile-quality/issues
+published_at: 2021-01-13T13:09:49.720612+01:00
+source:
+  git: https://github.com/imranMnts/bitrise-step-mobile-quality.git
+  commit: e025f9949279db5a640f419fe949e2687ceca3f8
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+- flutter
+- cordova
+- ionic
+- xamarin
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
Dependency management looks very messy.
You are declaring git which does not seem to be used.
You are using brew inside a script: https://github.com/imranMnts/bitrise-step-mobile-quality/blob/master/step.sh#L29 which is forbidden by the guidelines.

It seems that you haven't read the Step Development Guideline with understanding.

Reply:
Ah sorry, looks like I misunderstood this part yes, I fixed it now